pkg/helper/helpers.go

```diff
@@ -258,3 +258,13 @@ func ChunkSlice[T any](slice []T, chunkSize int) [][]T {
 func IsBYOVPC(cluster *cmv1.Cluster) bool {
 	return len(cluster.AWS().SubnetIDs()) > 0
 }
+
+func FilterEmpty(s []string) []string {
+	var r []string
+	for _, str := range s {
+		if str != "" {
+			r = append(r, str)
+		}
+	}
+	return r
+}
```

Comment:
NP: Can we avoid the single character naming in this func, I know its recommended, but something we tend to avoid across our codebase :pray:

Reply:
Sure thanks @ciaranRoche , will make the change

Reply:
Updated FYI @ciaranRoche